kickstarts/base.ks.erb

```diff
@@ -67,6 +67,13 @@ $appliance_root/cfme-setup.sh
 
 <%= render_partial "post/db_init" %>
 
+# make sure we have the device name in all ifcfg-* files
+ls /etc/sysconfig/network-scripts/ifcfg-* | while read FILE
+do
+  DEVICE=`echo $FILE | cut -d'/' -f5 | cut -d'-' -f2`
+  [[ -z "`grep 'DEVICE=' $FILE`" ]] && echo "DEVICE=$DEVICE" >> $FILE
```

Comment:
looks good

Protip 1:

I think you could use the return status from grep instead of `-z`. something like:

```bash
[[ grep -q 'DEVICE=' $FILE ]] || echo "DEVICE=$DEVICE" >> $FILE
```

protip 2:

I think you can get the device name using bash `DEVICE=${FILE##*-}`